axelrod/strategies/worse_and_worse.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+from axelrod import Actions, Player, random_choice
+
+C, D = Actions.C, Actions.D
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / 1000'. Therefore
+    it is more and more likely to defect as the round goes on.
```

Comment:
Would this line go at the end of line 10 or do you want it as part of the names section?